.github/workflows/ci.yml

```diff
@@ -288,87 +221,29 @@ jobs:
           name: coverage-html
           path: htmlcov
 
-  test-fastapi:
-    name: test fastAPI
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v3
-
-    - name: set up python
-      uses: actions/setup-python@v4
-      with:
-        python-version: '3.10'
-
-    - name: install
-      run: make install-testing
-
-    - name: test
-      run: make test-fastapi
-
-  build:
-    name: build py3.${{ matrix.python-version }} on ${{ matrix.platform || matrix.os }}
-    needs: [lint, test-linux-compiled, test-not-compiled, test-old-mypy, test-fastapi]
-    if: "success() && (startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/main')"
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [ubuntu , macos , windows]
-        python-version: ['7', '8', '9', '10', '11']
-        include:
-        - os: ubuntu
-          platform: linux
-        - os: windows
-          ls: dir
-
-    runs-on: ${{ matrix.os }}-latest
-    steps:
-    - uses: actions/checkout@v3
-
-    - name: set up python
-      uses: actions/setup-python@v4
-      with:
-        python-version: '3.8'
-
-    - name: install
-      run: pip install -U twine setuptools wheel cibuildwheel
-
-    - name: build sdist
-      if: matrix.os == 'ubuntu' && matrix.python-version == '9'
-      run: python setup.py sdist bdist_wheel
-      env:
-        SKIP_CYTHON: 1
-
-    - name: build ${{ matrix.platform || matrix.os }} binaries
-      run: cibuildwheel --output-dir dist
-      env:
-        PIP: 'pip'
-        CIBW_BUILD: 'cp3${{ matrix.python-version }}-*'
-        CIBW_SKIP: '*-win32'
-        CIBW_PLATFORM: '${{ matrix.platform || matrix.os }}'
-        CIBW_BEFORE_BUILD: 'pip install -U cython'
-        CIBW_TEST_REQUIRES: 'pytest==6.2.5 pytest-mock==3.6.1'
-        CIBW_TEST_COMMAND: 'pytest {project}/tests'
-        CIBW_MANYLINUX_X86_64_IMAGE: 'manylinux2014'
-        CIBW_MANYLINUX_I686_IMAGE: 'manylinux2014'
-        CIBW_ARCHS_MACOS: 'x86_64 arm64'
-        CIBW_TEST_SKIP: '*-macosx_arm64'  # see https://cibuildwheel.readthedocs.io/en/stable/faq/#universal2
-
-    # TODO build windows 32bit binaries
-
-    - name: list dist files
-      run: |
-        ${{ matrix.ls || 'ls -lh' }} dist/
-        twine check dist/*
-
-    - name: Store dist artifacts
-      uses: actions/upload-artifact@v3
-      with:
-        name: pypi_files
-        path: dist
+# FastAPI is has a version constraint pydantic<2.0.0, so we can't run tests, we expect them to break for now anyway
```

Comment:
@tiangolo I'm commenting out these tests at the moment because:
1. fastapi requires pydantc<2 so this either fails or tests pydantic v1.10 which is pointless
2. They'll start failing soon since fastapi won't support pydantic 2 when it's in development

Hope that's okay with you, if you'd like us to test FastAPI, happy to support it somehow.